--- kismet-2010-07-R1/conf/kismet_drone.conf~	2008-12-11 00:12:27.000000000 +0200
+++ kismet-2010-07-R1/conf/kismet_drone.conf	2011-01-13 20:03:48.696974538 +0200
@@ -16,7 +16,7 @@
 droneringlen=65535
 
 # Do we have a GPS?
-gps=true
+gps=false
 # Do we use a locally serial attached GPS, or use a gpsd server?
 # (Pick only one)
 gpstype=gpsd
--- kismet-2016-07-R1/conf/kismet.conf.in~	2016-07-02 20:06:10.000000000 +0200
+++ kismet-2016-07-R1/conf/kismet.conf.in	2017-01-17 10:59:15.329493490 +0100
@@ -11,7 +11,7 @@
 # servername=Kismet Server
 
 # Prefix of where we log (as used in the logtemplate later)
-# logprefix=/some/path/to/logs
+logprefix=@localstatedir@/log/kismet
 
 # Do we process the contents of data frames?  If this is enabled, data
 # frames will be truncated to the headers only immediately after frame type
@@ -122,13 +122,13 @@
 # OUI file, expected format 00:11:22<tab>manufname
 # IEEE OUI file used to look up manufacturer info.  We default to the
 # wireshark one since most people have that.
-ouifile=/etc/manuf
-ouifile=/usr/share/wireshark/wireshark/manuf
+#ouifile=/etc/manuf
+#ouifile=/usr/share/wireshark/wireshark/manuf
 ouifile=/usr/share/wireshark/manuf
-ouifile=/Applications/Wireshark.app/Contents/Resources/share/wireshark/manuf
+#ouifile=/Applications/Wireshark.app/Contents/Resources/share/wireshark/manuf
 
 # Do we have a GPS?
-gps=true
+gps=false
 # Do we use a locally serial attached GPS, or use a gpsd server, or
 # use a fixed virtual gps?
 # (Pick only one)
