--- kismet-2021-05-R1/conf/kismet_logging.conf~	2021-05-20 00:58:52.000000000 +0200
+++ kismet-2021-05-R1/conf/kismet_logging.conf	2021-05-23 13:45:39.393458522 +0200
@@ -14,7 +14,7 @@
 # in, but the logs can be automatically stored in other directories as well.
 # The directory must exist before Kismet is started - Kismet will /not/ create
 # the directory list itself.
-log_prefix=./
+log_prefix=/var/log/kismet
 
 
 # Logging is enabled by type; plugins may add additional types.  The log types can be
