diff -urN kismet-2.4.6.org/pcapsource.cc kismet-2.4.6/pcapsource.cc
--- kismet-2.4.6.org/pcapsource.cc	Wed Sep  4 18:02:17 2002
+++ kismet-2.4.6/pcapsource.cc	Wed Sep  4 18:03:04 2002
@@ -75,7 +75,7 @@
 
 #ifdef HAVE_PCAP_NONBLOCK
     pcap_setnonblock(pd, 1, errstr);
-#elseif ! defined(SYS_OPENBSD)
+#elif !defined(SYS_OPENBSD)
     // do something clever  (Thanks to Guy Harris for suggesting this).
     int save_mode = fcntl(pcap_fileno(pd), F_GETFL, 0);
     if (fcntl(pcap_fileno(pd), F_SETFL, save_mode | O_NONBLOCK) < 0) {
