--- kismet-2010-01-R1.orig/configure.in	2010-01-07 23:33:30.000000000 +0000
+++ kismet-2010-01-R1/configure.in	2010-01-15 03:40:30.000000000 +0000
@@ -849,7 +849,11 @@
 havenetlink=yes
 	AC_CHECK_HEADERS([asm/types.h netlink/genl/genl.h netlink/genl/family.h netlink/genl/ctrl.h netlink/msg.h netlink/attr.h linux/nl80211.h linux/if_arp.h linux/wireless.h], 
 	AC_DEFINE(HAVE_NETLINKHEADERS, 1, Netlink headers are there), 
-	havenetlink=no)
+	havenetlink=no,
+	[[#ifdef HAVE_SYS_SOCKET_H
+	 # include <sys/socket.h>
+	 #endif
+	 ]])
 
 
 #if test "$havenetlink" = "yes"; then
